sdk/identity/azure-identity-broker/README.md

```diff
@@ -1,12 +1,11 @@
-
-
 # Azure Identity Broker plugin for Python
 
-This package extends the [Azure Identity][azure_identity] library by providing supplemental credentials for authenticating via an authentication broker.
+This package extends the [Azure Identity][azure_identity] library by providing supplemental credentials for authenticating via an authentication broker. An authentication broker is an application that runs on a user's machine that manages the authentication handshakes and token maintenance for connected accounts. The table below outlines supported brokers and the minimum package version required to use each of them.
 
-An authentication broker is an application that runs on a user’s machine that manages the authentication handshakes and token maintenance for connected accounts. Currently, only the following brokers are supported:
-- Web Account Manager (WAM) on Windows
-- Company Portal on macOS
+| Broker                                    | Minimum package version |
+|-------------------------------------------|-------------------------|
+| [Company Portal][company_portal] on macOS | 1.3.0                   |
```

Comment:
1.3.0b1?

Reply:
I set to the GA version so we don't have to update it again. Does that approach work for you?

Reply:
Today, our plan is still we will keep it in preview until we have all languages support and there is no ETA for Java yet. IOW, the GA version will not be available at least in a few months. I am afraid it may confuse our customers.

Reply:
Knowing that, I'll update it to the beta version.